### DISCUSSION TITLE
Stat panel: Separate y-axis scale for each series graph

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64466] [Dashboards, Panels & Visualizations]
I would like to be able to have each series in a single panel displayed with independent y-axis scaling. Right now, I get what is shown in the top row here if I put all series into a single panel. What I want is what is on the bottom row, but I want it in a single panel rather than individual panels:
![image](https://user-images.githubusercontent.com/7028403/223829996-14904a4a-3815-4e23-baf0-8f5382dd5de2.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82831. Feel free to continue the discussion around this there. Thank you!
